lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -426,6 +426,16 @@ func buildCreate(op trace.Operation, d *data.Data, finder *find.Finder, vch *mod
 
 func handleCreate(op trace.Operation, c *create.Create, validator *validate.Validator) (*strfmt.URI, error) {
 	vchConfig, err := validator.Validate(validator.Context, c.Data)
+	if err != nil {
+		issues := validator.GetIssues()
+		messages := make([]string, 0, len(issues))
+		for _, issue := range issues {
```

Comment:
just a suggestion: will it be neater if we move the logic of appending multiple errors to `util`? issues is just a list of `error`

Reply:
I think my preference would be to leave the functionality where it is for now, and refactor when we actually have the second use case.

I suspect we'll want to deduplicate a lot of logic between VCH creation and VCH re-configuration, but I don't want to presuppose any specific architecture.